ui/css/loading.scss

```diff
@@ -11,3 +13,50 @@
   align-self: center;
   margin-top: 1rem;
 }
+
+.loading-timeout-message {
+  align-self: center;
+  animation: showTimeoutMessage 20s ease-in-out forwards;
+  color: var(--color-text-alternative);
+  margin-top: 4rem;
+  max-width: design-system.$screen-sm-max;
+  opacity: 0;
+  text-align: center;
+
+  &::before {
+    direction: "__MSG_@@bidi_dir__";
```

Comment:
This is a "magic" MSG string that is intended to be used exactly for this scenario. It determines if the language should be rendered left-to-right or right-to-left.